tests/test_visitors/test_wrong_name/test_variable_names.py

```diff
@@ -17,9 +17,6 @@
 
 underscore_variable_test = """
 _{0} = 'test'
-{0}_ = 'test'
```

Comment:
Test still should check these patterns, please, do not remove them.

Reply:
@sobolevn Unfortunately, i'can't add them in one string due to how tests work